lib/abstract_format.ml

```diff
@@ -23,13 +23,14 @@ and form_t =
   | ModDecl of form_t list
 
   | AttrExport of line_t * (string * int) list
+  | AttrExportType of line_t * (string * int) list
   | AttrImport of line_t * (string * int) list
   | AttrMod of line_t * string
   | AttrFile of line_t * string * line_t
   | DeclFun of line_t * string * int * clause_t list
   | SpecFun of line_t * string option * string * int * type_t list
   | DeclRecord of line_t * (line_t * string * expr_t option * type_t option) list
-  | DeclType
+  | DeclType of line_t * bool * string * (line_t * string) list * type_t
```

Comment:
NOTE: A `bool` parameter at the 2nd position becomes `true` when the declaration is a `opaque`, right?

Reply:
Yes, you're right.
The second parameter means something like `is_opaque`.

Reply:
If so, would you add a comment about  a meaning of that or use a data type (e.g. `AttrType | AttrOpacue`) to clarify the meaning, please? Do you have any ideas?